### ISSUE TITLE
Update vue-test-utils usage

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4-vue#86]
## Are you reporting a feature request or a bug?

Task

## Provide detailed reproduction steps (if any)

1. Run `npm run test`

### Expected result

Test are passing or failing without deprecate warnings.

### Actual result

```
ERROR: '[vue-test-utils]: overwriting methods via the `methods` property is deprecated and will be removed in the next major version. There is no clear migration path for the `methods` property - Vue does not support arbitrarily replacement of methods, nor should VTU. To stub a complex method extract it from the component and test it in isolation. Otherwise, the suggestion is to rethink those tests.'
ERROR: '[vue-test-utils]: options.attachToDocument is deprecated in favor of options.attachTo and will be removed in a future release is deprecated and will be removed in the next major version.'
```

## Other details

* Browser: IE11
* OS: Linux Ubuntu
* Integration version: 
* CKEditor version: 1.2.0
* Installed CKEditor plugins: -

**Comment by sculpt0r** [comment]
Fixing https://github.com/ckeditor/ckeditor4-vue/issues/124 - may be a good moment to try to bump this dependency - at least above the beta release.

**Comment by sculpt0r** [comment]
Closed in #128 - vtu bumped above the beta version.
